src/index.js

```diff
@@ -9,7 +9,7 @@ const lodash = require('lodash');
  * @param  {integer}  ship
  * @return  {bool}  true if galaxy, false otherwise
  */
-const isGalaxy = (ship) => Number.isInteger(ship) && ship >= 0 && ship < 256;
+const isGalaxy = ship => Number.isInteger(ship) && ship >= 0 && ship < 256;
```

Comment:
> remove parens around single arg to arrow func

Strongly not in favor of this, because consistency, clarity, etc. It's like `;` or `{}` being optional sometimes, you should just always use them to make sure you don't trip yourself up.

Usual disclaimer that I'm not a JS dev. Might want to hear what @c-johnson has to say about this.

Reply:
Whatever @chris says goes, strong opinions held loosely etc

Reply:
My 2c is that this isn't unusual, but I understand why it might be confusing to 'non-js-people' blessed as they may be.

Reply:
FWIW, I think leaving off the parens for single-arg arrow functions is idiomatic JS.